PhysicsTools/PatAlgos/python/producersLayer1/jetUpdater_cfi.py

```diff
@@ -1,6 +1,6 @@
 import FWCore.ParameterSet.Config as cms
 
-patJetsUpdated = cms.EDProducer("PATJetUpdater",
+updatedPatJets = cms.EDProducer("PATJetUpdater",
```

Comment:
Yes, with the updateJetCollection() function from jetTools.py, this becomes a transient collection. Also, the new name is more in line with the PAT jet naming convention where user-defined labels and postfixes come after "PatJets" while "updated" is more a prefix-type descriptor.